include/SDL3/SDL_gpu.h

```diff
@@ -44,49 +44,256 @@ extern "C" {
+ */
 typedef struct SDL_GPUFence SDL_GPUFence;
 
+/**
+ * Specifies the primitive topology of a graphics pipeline.
+ *
+ * \since This enum is available since SDL 3.0.0
+ *
+ * \sa SDL_CreateGPUGraphicsPipeline
+ */
 typedef enum SDL_GPUPrimitiveType
 {
-    SDL_GPU_PRIMITIVETYPE_POINTLIST,
-    SDL_GPU_PRIMITIVETYPE_LINELIST,
-    SDL_GPU_PRIMITIVETYPE_LINESTRIP,
-    SDL_GPU_PRIMITIVETYPE_TRIANGLELIST,
-    SDL_GPU_PRIMITIVETYPE_TRIANGLESTRIP
+    SDL_GPU_PRIMITIVETYPE_POINTLIST,     /**< A series of separate points. */
+    SDL_GPU_PRIMITIVETYPE_LINELIST,      /**< A series of separate lines. */
+    SDL_GPU_PRIMITIVETYPE_LINESTRIP,     /**< A series of connected lines. */
+    SDL_GPU_PRIMITIVETYPE_TRIANGLELIST,  /**< A series of separate triangles. */
+    SDL_GPU_PRIMITIVETYPE_TRIANGLESTRIP  /**< A series of connected triangles. */
 } SDL_GPUPrimitiveType;
 
+/**
+ * Specifies how the contents of a texture attached to a render pass are treated at the beginning of the render pass.
+ *
+ * \since This enum is available since SDL 3.0.0
+ *
+ * \sa SDL_BeginGPURenderPass
+ */
 typedef enum SDL_GPULoadOp
 {
-    SDL_GPU_LOADOP_LOAD,
-    SDL_GPU_LOADOP_CLEAR,
-    SDL_GPU_LOADOP_DONT_CARE
+    SDL_GPU_LOADOP_LOAD,      /**< The previous contents of the texture will be preserved. */
+    SDL_GPU_LOADOP_CLEAR,     /**< The contents of the texture will be cleared to a color. */
+    SDL_GPU_LOADOP_DONT_CARE  /**< The previous contents of the texture need not be preserved. The contents will be undefined. */
 } SDL_GPULoadOp;
 
+/**
+ * Specifies how the contents of a texture attached to a render pass are treated at the end of the render pass.
+ *
+ * \since This enum is available since SDL 3.0.0
+ *
+ * \sa SDL_BeginGPURenderPass
+ */
 typedef enum SDL_GPUStoreOp
 {
-    SDL_GPU_STOREOP_STORE,
-    SDL_GPU_STOREOP_DONT_CARE
+    SDL_GPU_STOREOP_STORE,     /**< The contents generated during the render pass will be written to memory. */
+    SDL_GPU_STOREOP_DONT_CARE  /**< The contents generated during the render pass are not needed and may be discarded. The contents will be undefined. */
 } SDL_GPUStoreOp;
 
+/**
+ * Specifies the size of elements in an index buffer.
+ *
+ * \since This enum is available since SDL 3.0.0
+ *
+ * \sa SDL_CreateGPUGraphicsPipeline
+ */
 typedef enum SDL_GPUIndexElementSize
 {
-    SDL_GPU_INDEXELEMENTSIZE_16BIT,
-    SDL_GPU_INDEXELEMENTSIZE_32BIT
+    SDL_GPU_INDEXELEMENTSIZE_16BIT, /**< The index elements are 16-bit. */
+    SDL_GPU_INDEXELEMENTSIZE_32BIT  /**< The index elements are 32-bit. */
 } SDL_GPUIndexElementSize;
 
-/* Texture format support varies depending on driver, hardware, and usage flags.
+/**
+ * Specifies the pixel format of a texture.
+ *
+ * Texture format support varies depending on driver, hardware, and usage flags.
  * In general, you should use SDL_GPUTextureSupportsFormat to query if a format
  * is supported before using it. However, there are a few guaranteed formats.
  *
```

Comment:
Whoops, this got mangled by the wiki bridge. You need a newline between "supported:" and the list of formats.